hack/processing-plan.yaml

```diff
@@ -580,6 +580,19 @@ files:
       - kustomization.yaml
     continueOnError: true
     operations:
+      - type: DELETE
+        key: .spec.template.spec.containers[0].env
+      - type: DELETE
```

Comment:
The same question here. It looks like much easier just update instead of delete and add on postOperations.